README.md

```diff
@@ -6,6 +6,8 @@
 
 This plugin decorates a Fastify instance with `routes`, which is a [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) of registered routes. Note that you have to register this plugin before registering any routes so that it can collect all of them.
 
+Note also that current versions of Fastify define plugin registration as async, so you have to ensure that registration completes before creating any routes.  See the examples below.
```

Comment:
I dont like the wording and it is confusing. 

The previous sentence says it all "Note that you have to register this plugin before registering any routes so that it can collect all of them." 

Either it is clear, that the sentene expresses that registering means that fastify-routes needs to be fully loaded before adding routes OR we need to clarify that sentence.

Reply:
Clearly it is not clear enough, since we are still having misunderstanding here.  The issue is that calling register ahead of registering the handlers isn't sufficient (any more) since you need to wait for it to finish.  Existing documentation implies you just need to call register(...) first so it is quite important to call attention to the difference between just calling it versus waiting for it.

Reply:
I agree btw that probably the right way to resolve it is just to rewrite the first part to include the explanation that you have to wait for it, not just call it.

Reply:
@Fdawgs 
@jsumners 

Do you have suggestions for a technical clear sentence? please and thank you ;)